apps/src/lib/ui/accounts/ChangeUserTypeModal.jsx

```diff
@@ -0,0 +1,158 @@
+import React, {PropTypes} from 'react';
+import i18n from '@cdo/locale';
+import {hashEmail} from '../../../code-studio/hashEmail';
+import BaseDialog from '../../../templates/BaseDialog';
+import color from '../../../util/color';
+import {isEmail} from '../../../util/formatValidation';
+import {Header, ConfirmCancelFooter} from '../SystemDialog/SystemDialog';
+import ChangeUserTypeForm from './ChangeUserTypeForm';
+
+/*
+
+Note: This feature is in active development, so there are still some rough edges.
```

Comment:
Any known issues worth mentioning?

Reply:
Good question.  I feel like this is pretty polished now and maybe I should update this comment to not mention rough edges. It _is_ still in active development though, because there's a good bit of follow-up work to add the email opt-in to both dropdowns.